ci/build_wheel.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name=$1
+package_dir=$2
+
+source rapids-configure-sccache
+source rapids-date-string
+
+# Use gha-tools rapids-pip-wheel-version to generate wheel version then
+# update the necessary files
+version_override="$(rapids-pip-wheel-version ${RAPIDS_DATE_STRING})"
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+
+ci/release/apply_wheel_modifications.sh ${version_override} "-${RAPIDS_PY_CUDA_SUFFIX}"
+echo "The package name and/or version was modified in the package source. The git diff is:"
+git diff
+
+cd "${package_dir}"
+
+# Hardcode the output dir
+python -m pip wheel . -w dist -vvv --no-deps --disable-pip-version-check
```

Comment:
```suggestion
python -m pip wheel . -w dist --no-deps --disable-pip-version-check
```

Is the verbosity here necessary?

In the past, we've intentionally turned off verbosity for things like `pytest` because it can make the GHA logs difficult to read.

Reply:
I will let @vyasr comment on the verbosity questions

Reply:
Yes I was going to raise this point towards the end. I say let's turn off the verbosity for tests, but keep it for builds (i.e. `pip wheel -v`) That matches what we do for conda and I think that's generally helpful. Test failures tend to be self-contained, so it's enough to see that a single test failure to know what's going on, whereas with builds the real cause can often be hidden (e.g. pip does something wrong in dependency installation which results in compile-time errors).